src/entity/Alerts.ts

```diff
@@ -38,7 +38,7 @@ export class Alerts {
   lastBanner: Date | null;
 
   // Should not be exposed to the client
```

Comment:
Comment here is not longer needed?

Reply:
Hmm still valid I think, more a extra note :)

Reply:
Thought it was added because of `select` option?

Reply:
Our flags are private by default we use `*Flags` and `*FlagsPublic` to expose them through graphql usually.

Reply:
Ah I looked closely, `getAlerts` is used in `boot` which escapes graphql.